app/logic/events.py

```diff
@@ -189,9 +189,9 @@ def getUpcomingEventsForUser(user,asOf=datetime.datetime.now()):
     events = (Event.select(Event)
                             .join(ProgramEvent)
                             .join(Interest, on=(ProgramEvent.program == Interest.program))
-                            .where(Interest.user == user)
-                            .where(Event.startDate >= asOf)
-                            .where(Event.timeStart > asOf.time())
+                            .where(Interest.user == user,
+                            Event.startDate >= asOf,
+                            Event.timeStart > asOf.time())
```

Comment:
Let's line these up with `Interest`, not `.where`

Reply:
resolved